java.js

```diff
@@ -0,0 +1,24 @@
+const hamburger = document.querySelector('.menu');
+const mobileMenu = document.querySelector('.mobile-menu');
+const mobilelistlink = document.querySelectorAll('#mobilelist li a');
+const cancelBtn = document.querySelector('#cancel-btn');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.add('hide');
+  cancelBtn.classList.remove('hide');
+  mobileMenu.style.display = 'flex';
+});
+
+mobilelistlink.forEach((a) => {
+  a.addEventListener('click', () => {
+    mobileMenu.style.display = 'none';
+    cancelBtn.classList.add('hide');
+    hamburger.classList.remove('hide');
+  });
+});
+
+cancelBtn.addEventListener('click', () => {
+  cancelBtn.classList.add('hide');
+  hamburger.classList.remove('hide');
+  mobileMenu.style.display = 'none';
+});
```

Comment:
- [ ] Please use the `DRY` principle you are not using this principle in your code. `DRY` stands for `Don't repeat yourself` and is a principle of software development aimed at reducing the repetition of software patterns, replacing it with abstractions or using data normalization to avoid redundancy. So kindly implemented This  principle in your code :slightly_smiling_face: check below :point_down:  my suggestions :+1: 

 ```suggestion
const hamburger = document.querySelector('.menu');
const mobileMenu = document.querySelector('.mobile-menu');
const mobilelistlink = document.querySelectorAll('#mobilelist li a');
const cancelBtn = document.querySelector('#cancel-btn');

const toggleMobileMenu = () => {
  hamburger.classList.toggle('hide');
  cancelBtn.classList.toggle('hide');
  mobileMenu.style.display = mobileMenu.style.display === 'flex' ? 'none' : 'flex';
}

const hideMobileMenu = () => {
  hamburger.classList.remove('hide');
  cancelBtn.classList.add('hide');
  mobileMenu.style.display = 'none';
}

hamburger.addEventListener('click', toggleMobileMenu);

mobilelistlink.forEach((a) => {
  a.addEventListener('click', hideMobileMenu);
});

cancelBtn.addEventListener('click', hideMobileMenu);
```